packages/expo-camera/src/useWebQRScanner.ts

```diff
@@ -39,7 +39,7 @@ const qrWorkerMethod = ({ data, width, height }: ImageData): any => {
 function useRemoteJsQR() {
   return useWorker(qrWorkerMethod, {
     remoteDependencies: ['https://cdn.jsdelivr.net/npm/jsqr@1.2.0/dist/jsQR.min.js'],
-    timeout: 5000,
+    autoTerminate: false,
```

Comment:
Have you tested on an Android web browser and iOS web browser, the functionality is a bit different between the two which is why we implemented this heavy stuff.

Reply:
Tested on:

- Safary iOS 15.1
- Firefox Android 11
- Firefox Windows 10
- Chrome Windows 10

Everything seems ok.

What's the difference you are talking about?